ppx/test/example.ml

```diff
@@ -40,10 +40,6 @@ module Cases = struct
     C ({|["B", 42]|}, poly_of_json, poly_to_json, (`B 42 : poly));
     C ({|["Fix",["Fix",["Fix",["A"]]]]|}, recur_of_json, recur_to_json, (Fix (Fix (Fix A))));
     C ({|["Fix",["Fix",["Fix",["A"]]]]|}, polyrecur_of_json, polyrecur_to_json, (`Fix (`Fix (`Fix `A))));
-    C ({|"A"|}, evar_of_json, evar_to_json, (A : evar));
-    C ({|"b_aliased"|}, evar_of_json, evar_to_json, (B : evar)); (* variant B repr as "b_aliased" in JSON *)
-    C ({|"b"|}, epoly_of_json, epoly_to_json, (`b : epoly));
-    C ({|"A_aliased"|}, epoly_of_json, epoly_to_json, (`a : epoly)); (* polyvariant `a aliased to "A_aliased"*)
```

Comment:
no longer supported

Reply:
can't we keep the test cases and print the fixed representation, or were they redundant?